jkanban.js

```diff
@@ -54,8 +54,8 @@ var dragula = require('dragula');
       dragBoard: function (el, source) {},
       dragendBoard: function (el) {},
       dropBoard: function (el, target, source, sibling) {},
-      click: function (el) {},
-      context: function (el, e) {},
+      click: null,
```

Comment:
I would not recommend that. I guess it would be nice to have a kind of property or even a callback function to get if user wants to have native click or context.